session/bootstrap.go

```diff
@@ -1319,8 +1322,8 @@ type bindInfo struct {
 	source     string
 }
 
-func upgradeToVer61(s Session, ver int64) {
-	if ver >= version61 {
+func upgradeToVer66(s Session, ver int64) {
+	if ver >= version66 {
```

Comment:
version in https://github.com/pingcap/tidb/pull/23161/files#diff-87683a21082ba99b8f8ee16504246f92fc7a8e417374ee868d00dc6bc773b015L1359 and https://github.com/pingcap/tidb/pull/23161/files#diff-87683a21082ba99b8f8ee16504246f92fc7a8e417374ee868d00dc6bc773b015R1350 should be updated as well.